### DISCUSSION TITLE
Team Page V1

### DISCUSSION THREAD

**Discussion by eliasfaltin** [GECO-Project-App/facilitation-app#116] [2 - Shaping]
The app should have a team page.
In Version 1 of this page we need some things to support the "Tutorial to me" exercise I described in a separate Discussions post.

2 Buttons
- Join a team -> Text field where a user can input a team code "ABC 123" (this should happen automagically if a signed-in user clicks on an invite link. Invite links could work like this: app.projectgeco.com/signup/?abc-123
- Create a team
  - Team Name
  - Invite members
    - "email address" text field
    - invite another team member button (which adds another text field)

Once an account has at least one team associated with it, the teams page should have a dropdown menu at the top showing the team name. If you press on the team name, you see all other teams you are a member of plus a "create new team" button at the bottom of that list.

The team page should show a grid/gallery/list of team members in the team (Profile Picture + Name) and a "Add Team members" button (in the top right or so maybe?).

When clicking on a given team member, we show the results/output of the tutorial to me exercise.

Down the line, this view will evolve into a "Baseball card" for each team member showing their honors/feedback and other relevant info (personality type, skill inventory, etc.)

Later on, we might want to create a "Administration" tab to allow team facilitators to remove, add, and change access powers for members (like promoting them to "Facilitator", allowing that person to create agendas for meetings). We might also want to add an "Agreements" tab where we show the outcome of "What is our Purpose" exercises and such. I think of this as a type of Wiki for the team and project. Focused on purpose, strategy decisions, and "How we work" kind of content.

**Comment by aripuguh** [comment]
Thanks for this! Exciting about how the app would turn into something more engaging with such features! Exploring it today as well.

**Comment by aripuguh** [comment]
Latest team page flow is [here](https://www.figma.com/design/jA7uB2CX76FYBxRzwmTDzA/Facilitation-App?node-id=479-980) @eliasfaltin 
![Facilitation App](https://github.com/user-attachments/assets/7436843c-d238-4bf3-95ad-4fa2f457590c)
